client/src/utils/validators.ts

```diff
@@ -26,6 +26,11 @@ export const maxValue = (max: any) => (value: any) => {
   }
 };
 
+export const maxValue = (max: any) => (value: any) =>
+  !value || value?.length <= max
+    ? undefined
+    : `${MAX_VALUE_ERROR_MESSAGE} ${max}`;
+
```

Comment:
why you create new method for it, you can modified or split above method

Reply:
Resolved